lib/firefox/webdriver/firefoxPreferences.js

```diff
@@ -88,25 +88,6 @@ module.exports = {
   'browser.newtabpage.activity-stream.default.sites': '',
   'browser.newtabpage.activity-stream.telemetry': false,
   'browser.reader.detectedFirstArticle': true,
-  'browser.safebrowsing.blockedURIs.enabled': false,
-  'browser.safebrowsing.downloads.enabled': false,
-  'browser.safebrowsing.downloads.remote.url':
-    'http://127.0.0.1/safebrowsing-dummy/downloads',
-  'browser.safebrowsing.malware.enabled': false,
-  'browser.safebrowsing.passwords.enabled': false,
-  'browser.safebrowsing.phishing.enabled': false,
-  'browser.safebrowsing.provider.google.gethashURL':
-    'http://127.0.0.1/safebrowsing-dummy/gethash',
-  'browser.safebrowsing.provider.google.updateURL':
-    'http://127.0.0.1/safebrowsing-dummy/update',
-  'browser.safebrowsing.provider.google4.gethashURL':
-    'http://127.0.0.1/safebrowsing4-dummy/gethash',
-  'browser.safebrowsing.provider.google4.updateURL':
-    'http://127.0.0.1/safebrowsing4-dummy/update',
-  'browser.safebrowsing.provider.mozilla.gethashURL':
-    'http://127.0.0.1/safebrowsing-dummy/gethash',
-  'browser.safebrowsing.provider.mozilla.updateURL':
-    'http://127.0.0.1/safebrowsing-dummy/update',
```

Comment:
Independent of tracking protection, it's good that this was removed since Browsertime already had an option for disabling safebrowsing.

Reply:
Yeah I must have missed that when I merged.